test/karma.conf-ci.js

```diff
@@ -3,8 +3,22 @@ var uuid = require('uuid');
 
 module.exports = function(config) {
 
+  // These are the default coverageFiles. They should be updated when run from
+  // the privly-safari repo, https://github.com/privly/privly-safari
+  coverageFiles = {
```

Comment:
Whoops, missed this before, but I believe this should be `var coverageFiles = {`